tests/test_cmdLineFwk.py

```diff
@@ -762,7 +761,8 @@ def testSimpleQGraphReplaceRun(self):
         # changed)
         args.replace_run = True
         args.output_run = "output/run2"
-        fwk.runPipeline(copy.deepcopy(qgraph), taskFactory, args)
+        qgraph = fwk.makeGraph(self.pipeline, args)
```

Comment:
I made this change to demonstrate the fix but I wonder if we are punting to fixing this properly with unresolved ref removal that maybe I should not include this change here. @andy-slac would you rather this test failed once you remove unresolved refs or are you happy that this is the right fix for replace-run testing anyhow and so should keep this fix?

Reply:
I'm OK with this change. I guess once we have the ability to rewrite output run in a graph we will add a new unit test for that.